### DISCUSSION TITLE
[Feature] immich-cli commands for standard library updates

### DISCUSSION THREAD

**Discussion by diyoyo** [immich-app/immich#7458] [Feature Request]
### The feature

Hi, maybe I missed an episode and it already exists, but I've never used the cli because I thought it lacked all the basic features proposed in this web menu: 
<img width="997" alt="image" src="https://github.com/immich-app/immich/assets/6624639/9ff13be9-46c1-4891-821d-6a920653ccac">

I'm mostly intested in everything that comes after the "setting" button: 
Scanning new Library files -> Remove Offline Files

Something like `immich-cli -u target_user library::remove_offline` 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by pjft** [comment]
Came here to request this as well. The community project that does something similar to this either isn't working correctly right now, or just isn't reliable for some reason on my setup with an external library.
